tests/Doctrine/Migrations/Tests/Provider/OrmSchemaProviderTest.php

```diff
@@ -37,10 +36,14 @@ public function testCreateSchemaFetchesMetadataFromEntityManager(): void
         }
     }
 
-    public function testEntityManagerWithoutMetadataCausesError(): void
+    /**
+     * It should be OK to use migrations to manage tables not managed by
+     * the ORM.
+     *
+     * @doesNotPerformAssertions
```

Comment:
The tests should probably perform some assertion

Reply:
I had an "assertInstanceOf" but phpstan complained that this was already enforced by the return type of the method... :shrug:

Reply:
Do we expect the schema returned by `createSchema()` to be empty? Can we assert that?